persistent/_timestamp.c

```diff
@@ -185,8 +185,8 @@ TimeStamp_unpack(TimeStamp *self, TimeStampParts *p)
 {
     unsigned long v;
 
-    v = (self->data[0] * 16777216 + self->data[1] * 65536
-       + self->data[2] * 256 + self->data[3]);
+    v = (self->data[0] * 0x1000000 + self->data[1] * 0x10000
+       + self->data[2] * 0x100 + self->data[3]);
```

Comment:
Can we name all these constants in order to avoid typos?  (the majyk ones below, too).

Reply:
Any name suggestions? I wasn't doing any better than MASK_24 and the like, which didn't seem like much of an improvement.

Reply:
I think I may have it now. commit forthcoming.

Reply:
How about this?